### PR TITLE
Rename to ansible-core

### PR DESCRIPTION
**Pull request by relrod**
##### SUMMARY

First draft of s/ansible-base/ansible-core/g

##### ISSUE TYPE

- Bugfix Pull Request
- Test Pull Request

##### COMPONENT NAME

packaging

### CODE REVIEW

File: changelogs/fragments/default-test-container160.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 minor_changes:
-  - ansible-test - default container now uses default-test-container 2.7.0 and ansible-base-test-container 1.6.0. This brings in Python 3.9.0rc1 for testing.
+  - ansible-test - default container now uses default-test-container 2.7.0 and ansible-core-test-container 1.6.0. This brings in Python 3.9.0rc1 for testing.
```
Comment (sivel) [inline changelogs/fragments/default-test-container160.yml]: Guessing these changelog changes for ansible-base-test-container should be reverted

Comment (relrod) [reply changelogs/fragments/default-test-container160.yml]: I kept it on the premise that we're going to eventually rename the container anyway, but I can revert it for now.

Comment (relrod) [reply changelogs/fragments/default-test-container160.yml]: Done.

File: changelogs/fragments/ansible-base-update-containers.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 minor_changes:
   - ansible-test - OpenSuse container now uses Leap 15.2 (https://github.com/ansible/distro-test-containers/pull/48).
-  - ansible-test - Ubuntu containers as well as ``default-test-container`` and ``ansible-base-test-container`` are now slightly smaller due to apt cleanup (https://github.com/ansible/distro-test-containers/pull/46).
+  - ansible-test - Ubuntu containers as well as ``default-test-container`` and ``ansible-core-test-container`` are now slightly smaller due to apt cleanup (https://github.com/ansible/distro-test-containers/pull/46).
   - ansible-test - CentOS 8 container is now 8.2.2004 (https://github.com/ansible/distro-test-containers/pull/45).
-  - ansible-test - ``default-test-container`` and ``ansible-base-test-container`` now use Python 3.9.0 instead of 3.9.0rc1.
+  - ansible-test - ``default-test-container`` and ``ansible-core-test-container`` now use Python 3.9.0 instead of 3.9.0rc1.
```
Comment (sivel) [inline changelogs/fragments/ansible-base-update-containers.yml]: This one too.  When we rename the container, I imagine we'll create a new fragment that indicates the container was renamed.

Comment (relrod) [reply changelogs/fragments/ansible-base-update-containers.yml]: Fixed

**Review by sivel — APPROVED** [review]
shipit

**Pull request merged**